docs/spec/v1beta2/gitrepositories.md

```diff
@@ -152,7 +152,7 @@ data:
 #### SSH authentication
 
 To authenticate towards a Git repository over SSH, the referenced Secret is
-expected to contain `.data.identity` and `known_hosts` fields. With the respective
+expected to contain `identity` and `known_hosts` fields. With the respective
```

Comment:
Removed `.data.` to align with the approach with `known_hosts`. Alternatively, we could add the `.data.` prefix on both.

Reply:
I updated the example, but did not mention the `data`/`stringData` conundrum, as we support both and it will come down to user preference.

Also added a Flux CLI example for SSH secret creation. PTAL